react-portal-employees/src/components/navBar.tsx

```diff
@@ -126,47 +139,48 @@ const ResponsiveAppBar = () => {
                 </MenuItem>
               </Box>
 
-              {isLoggedIn ? (
-                <Box sx={{ flexGrow: 0, display: 'flex' }}>
+              {Cookies.get("user_session") ? (

```

Comment:
make a named method for this

Reply:
moved it to userMenu.tsx component and made as named method (line 69 in userMenu.tsx)